app/(main)/settings/team/actions.ts

```diff
@@ -1,10 +1,39 @@
 "use server";
 
-import { db, supabaseUserMappings, teamMemberships, teams } from "@/drizzle";
+import {
+	type TeamRole,
+	db,
+	supabaseUserMappings,
+	teamMemberships,
+	teams,
+	users,
+} from "@/drizzle";
 import { getUser } from "@/lib/supabase";
+import { fetchCurrentTeam, isProPlan } from "@/services/teams";
 import { and, eq } from "drizzle-orm";
 import { revalidatePath } from "next/cache";
 
+function isTeamRole(role: string): role is TeamRole {
+	return role === "admin" || role === "member";
+}
+
+export async function getTeamName() {
+	const user = await getUser();
+
+	// TODO: In the future, this query will be changed to retrieve from the selected team ID
+	const _teams = await db
+		.select({ dbId: teams.dbId, name: teams.name })
+		.from(teams)
+		.innerJoin(teamMemberships, eq(teams.dbId, teamMemberships.teamDbId))
+		.innerJoin(
+			supabaseUserMappings,
+			eq(teamMemberships.userDbId, supabaseUserMappings.userDbId),
+		)
+		.where(eq(supabaseUserMappings.supabaseUserId, user.id));
+
+	return _teams[0].name;
+}
+
```

Comment:
This can be removed.

```suggestion
```

I've removed this on my pull request.
https://github.com/giselles-ai/giselle/pull/163/files#diff-fd9634083caa6efe56527cb4a5c08420cc90650a15640a08396d234d38c29eebL14

Reply:
[9cf6781](https://github.com/giselles-ai/giselle/pull/135/commits/9cf6781aaf8f150462c3b2aade9dc695614db330) 
removed ✂️